tests/unit/test_sizes.py

```diff
@@ -120,54 +120,56 @@ def test_grid_parameters_from_harmony_message_has_complete_information(self):
         self.assertDictEqual(expected_parameters, actual_parameters)
 
     def test_grid_parameters_from_harmony_no_message_information(self):
-        """input granule is in preferred_crs on a 25km grid
-
-        But has grid extents different from the ICD defaults.
-
-        """
+        """input granule is in preferred_crs on a 25km grid"""
+        crs = CRS.from_epsg(sp_seaice_grid['epsg'])
+        height = sp_seaice_grid['height']
+        width = sp_seaice_grid['width']
+        img_transform = Affine.translation(
+            sp_seaice_grid['left'], sp_seaice_grid['top']
+        ) * Affine.scale(sp_seaice_grid['xres'], -1 * sp_seaice_grid['yres'])
 
-        preferred_scale_extent = {
-            'xmin': -4194304.0,
-            'ymin': -4194304.0,
-            'xmax': 4194304.0,
-            'ymax': 4194304.0,
+        left, bottom, right, top = rasterio.transform.array_bounds(
+            height, width, img_transform
+        )
+        image_scale_extent = {
```

Comment:
Super minor note of appreciation: Thanks for updating the variable name here to reflect what this now is.